package.json

```diff
@@ -25,6 +25,7 @@
     "example:api:dev": "pnpm run --filter \"api\" tauri dev"
   },
   "devDependencies": {
+    "@changesets/cli": "^2.29.7",
```

Comment:
We have this setup in the CI (also we use convector not changesets), please remove this